tsl/src/fdw/relinfo.c

```diff
@@ -81,10 +81,15 @@ apply_fdw_and_server_options(TsFdwRelInfo *fpinfo)
 TsFdwRelInfo *
 fdw_relinfo_get(RelOptInfo *rel)
 {
-	TimescaleDBPrivate *rel_private = rel->fdw_private;
+	TimescaleDBPrivate *rel_private;
+
+	if (!rel->fdw_private)
+		rel->fdw_private = ts_create_private_reloptinfo(rel);
```

Comment:
`ts_create_private_reloptinfo()` appears to set `rel->fdw_private` itself, so it's not needed to assign `rel->fdw_private =`.